core/commands/partials/terraform_configurator.rb

```diff
@@ -70,14 +71,14 @@ def node_provisioned?(node)
   def configure_with_chef(node, logger)
```

Comment:
This method is similar to one found in the Vagrant configurator. We should consider to extract them too. Not in this refactoring session.